### DISCUSSION TITLE
Update 6.10 - dev.2

### DISCUSSION THREAD

**Discussion by TBog** [TBog/TBLauncher#261] [Pre-Release]
* fix dock not showing when only visible for results
* add margin preferences for dock, search bar, result list
* remove unused resources
* add icon picker from local storage for tags and shortcuts
* removed okhttp3 dependency from icon picker

<hr /><em>This discussion was created from the release <a href='https://github.com/TBog/TBLauncher/releases/tag/v6.10-dev.2'>Update 6.10 - dev.2</a>.</em>

**Comment by ltguillaume** [comment]
Margins work great. As said, maybe considering a spacing parameter for the space between the individual results would be worthwhile.

**Comment by ltguillaume** [comment]
It seems that the option "Back action when closing keyboard" stopped working: when pressing the Back key to close the keyboard, only the keyboard closes, but the Back action requires another Back-press.

**Reply by TBog** [reply]
I can't reproduce it.

**Reply by ltguillaume** [reply]
Behavior:
- [x] Link keyboard to search bar visibility
- [x] "Back" action when closing keyboard
---
Desktop mode "Search":
- [x] Show dock
- Back button : Show "Widget" desktop

Android 10

**Reply by TBog** [reply]
After working on issue [#238 Back key in Search_Desktop is applied twice](https://github.com/TBog/TBLauncher/issues/238) I made some changes that probably don't work on your device. Are you able to reproduce your issue on a different device?

**Reply by ltguillaume** [reply]
It's a bit strange:
- Hardware Back button doesn't trigger the "Back" action (only closes keyboard)
- A Back gesture via Smart Task Launcher (using Android's Accessibility -> Full control) _does_ trigger the "Back" action (and closes the keyboard)

I'll try on a different device later.

**Reply by ltguillaume** [reply]
Ok, there's more going on now. The desired behavior (close keyboard + go back) worked on a Nokia 3, but with `"Back" action when closing keyboard` disabled, the Back button didn't do anything anymore: it's not closing the keyboard either. Is this the desired behavior now?

Tested with 6.11
